scripts/configmap/create_cm.sh

```diff
@@ -1,12 +1,164 @@
-k delete cm powerscale-driver-v2.2.0,driver-sidecars,authorization-v2.2.0,authorization-common -n dell-csm-operator
-kubectl create cm powerscale-driver-v2.2.0 --from-file=powerscale/v2.2.0/ -n dell-csm-operator
-kubectl get cm powerscale-driver-v2.2.0 -n dell-csm-operator
+#!/bin/bash
 
-kubectl create cm driver-sidecars --from-file=sidecars/ -n dell-csm-operator
-k get cm driver-sidecars -n dell-csm-operator
+source ../common.bash
 
-kubectl create cm authorization-v1.2.0 --from-file=authorization/v1.2.0 -n dell-csm-operator
-k get cm authorization-v2.2.0 -n dell-csm-operator
+NS="dell-csm-operator"
+DRIVER="powerscale"
+CONFIGVERSION="v2.3.0"
+# All below variables not recommended for user modification
+DELETE_EXIST_CFGMAP=0
+TEMPDIR="/tmp/csm-oper-cfg-map/"
+PROG="${0}"
+CFGMAP_NAMES_FILE="cm_names.txt"
+SCRIPTDIR=`pwd`
+cfgmaps_created=""
+
+function cleanup() {
+  # Get rid of temp directory
+  rm -rf $TEMPDIR
+  cd $SCRIPTDIR
+}
+
+function cleanup_created_cfgmaps() {
+  if [ "$cfgmaps_created" != "" ]; then
+    kubectl delete configmap $cfgmaps_created -n $NS
+  fi
+}
+
+function read_configmap_names() {
+  # make sure there is no existing list
+  rm -rf $CFGMAP_NAMES_FILE
+  log section "Read configmap names from file"
+  cat $DRIVER-$CONFIGVERSION.yaml | while read line
+  do
+    echo $line | grep listOfConfigMapNames
+      if [ "$?" == "0" ]; then
+        echo "reading configmaps list"
+          while read line
+          do
+            echo $line | grep "-"
+            if [ "$?" == "0" ]; then
+              echo $line | awk ' {print $2}' >> $CFGMAP_NAMES_FILE
+            else
+              log step "Done reading configmap names"
+              break
+            fi
+          done
+        fi
+  done
+}
+
+function usage() {
+  echo
```

Comment:
echo "this utiltity does xx yy" please add one liner on what this script does and when its to be used